modules/learn/pages/security/authentication-domains.adoc

```diff
@@ -167,3 +170,25 @@ Note that use of the PAM Linux password-module requires all cluster-nodes to be
 Additionally, the `saslauthd` library version must be 2.1.x or above.
 
 For details on configuring `saslauthd` to support external authentication by PAM, see xref:manage:manage-security/configure-saslauthd.adoc[Configure `saslauthd`].
+
+[#saml-authentication]
+== SAML Authentication
+
+SAML is a standard that defines how services communicate about user authentication. When an Administrator enables SAML authentication in Couchbase Server, a button appears on the Couchbase Server Web Console login page. 
+
+image::security/sso-login.png[,720]
+
+This button sends users to an _Identity Provider_ (IdP)'s site to authenticate. If the user authenticates with the IdP, the IdP sends the user back to Couchbase Server with an message containing their authentication information. Couchbase Server then logs the user into the Web Console. Users can only use SAML authentication to log into the Couchbase Web Console. They cannot authenticate for the command line, REST API, or SDK using SAML.
+
+SAML authentication offers these features:
+
+* _Single Sign On_ (SSO) lets users avoid repeated username and password prompts for every service they sign into. Instead, the user authenticates once with the IdP when first connecting to a service. This initial authentication starts a session for the user with the IdP. While the session is active, when the user connects to a service that uses the IdP, it authenticates them without prompting for a username and password. 
+* Most IdPs implement _Two-Factor Authentication_ (2FA). 2FA increases security by requiring users to prove their identity with more than just a username and password. For example, 2FA can require users to enter a number sent to them via a mobile text message or from an authentication app on their mobile phone. 
+* By centralizing user authentication, SAML reduces the workload of administrators. They have a single place to authorize new users and deauthorize users who should no longer have access. When an administrator deauthorizes a user, services automatically end all of the user's active sessions. This feature also benefits users by reducing the number of passwords they need to remember.
+
+Enabling SAML authentication does not prevent users who have locally defined accounts from logging into Couchbase Server directly with their username and password. In addition, you can enable both SAML authentication and LDAP authentication. However, users may become confused if you do not synchronize user names and passwords between the LDAP server and the IdP. 
+
+=== Mapping of Groups and Roles With SAML Authentication
```

Comment:
I think the main point I wanted to make in the paragraph was that enabling SAML authentication doesn't disable locally-defined users logging in via the usual username and password. I revised the paragraph and moved it to the end of the next section. I also added your suggested clarification of how the user's roles are determined, and statement that admins can manage users in the IdP.

Reply:
My comment was about the "Mapping of Groups and Roles With SAML Authentication" paragraph (not about 'Enabling SAML authentication does not prevent users'). Github doesn't let me highlight the paragraph when I am adding a comment so I am attaching the comment to the very beginning of the sentence. Now I realize that it can look like the comment is for the paragraph above. Sorry for confusion.

Reply:
I believe I've revised this in line with your comments.